src/main/dialogs/quickSetStrip/qssDialog.js

```diff
@@ -191,13 +191,21 @@ gpii.app.qss.computeQssWidth = function (buttonWidth, sideMargin, qssButtons) {
 
 /**
  * Handles the blur event for the QSS which is fired when the window loses focus. The QSS
- * will be closed only if this is the user's preference.
+ * will be closed only if this is the user's preference AND if the blur event was not the
+ * result of the user clicking the tray icon (i.e. the mouse pointer is not within the
+ * bounds of the tray icon).
  * @param {Component} that - The `gpii.app.qss` instance.
+ * @param {Component} tray - The `gpii.app.tray` instance.
  * @param {Boolean} closeQssOnBlur - If `true`, the QSS will be hidden once the window
  * loses focus. Otherwise, it will stay open.
  */
-gpii.app.qss.handleBlur = function (that, closeQssOnBlur) {
+gpii.app.qss.handleBlur = function (that, tray, closeQssOnBlur) {
     if (closeQssOnBlur) {
-        that.hide();
+        var trayBounds = tray.tray.getBounds(),
```

Comment:
It's an Electron method - [getBounds](https://electronjs.org/docs/api/tray#traygetbounds-macos-windows)

Reply:
Interesting, I'm getting a:
```
TypeError: Cannot read property 'getBounds' of undefined
    at gpii.app.qss.handleBlur (V:\src\main\dialogs\quickSetStrip\qssDialog.js:206:36)
    at fluid.componentConstructor.invokeInvoker [as handleBlur] (V:\node_modules\infusion\src\framework\core\js\FluidIoC.js:1742:29)
    at gpii.app.blurrable.onBlur (V:\src\main\dialogs\basic\blurrable.js:95:14)
    at togo (V:\node_modules\infusion\src\framework\core\js\FluidIoC.js:1806:33)
    at togo (V:\node_modules\infusion\src\framework\core\js\FluidIoC.js:1806:33)
    at fire (V:\node_modules\infusion\src\framework\core\js\Fluid.js:1568:40)
    at BrowserWindow.firer.fire (V:\node_modules\infusion\src\framework\core\js\FluidIoC.js:1934:40)
    at BrowserWindow.emit (events.js:187:15)
```

Reply:
Okay, this might be coming from Steve's TrayButton code, I'll let him know.

Reply:
That's strange, we don't have the Steve's trayButton in this PR. When do you experience this exception?

Reply:
@stegru @javihernandez 
We haven't observed any problems with the tray icon hiding in this PR.
 
The problem seems to be coming from the "Tray Button" implementation as this part of the code is missing there (since it is part of the current PR).
From what I see, for the `tray icon`, there's a renaming of the `tray` component member to `trayIcon`. Using `tray.trayIcon.getBounds()` should make things work for the `tray icon`.
 
Regarding the `tray button`, I can't see whether there's an API for getting the tray button position, so adding one should do the trick for the tray button as well.
 
As the `tray` component can be both `trayIcon` and `trayButton`, probably it would be the easiest if a generic interface, such as a `getTrayBounds` invoker, is added to it that would return a list of the screen positions of both the `trayIcon` and `trayButton`. Then the QS should be closed in case the cursor is not positioned in either of them.
 
I think it would be best if this PR is merged first and this issue is fixed with the `Tray Button` implementation.